src/main/java/Duke/Tasks/Find.java

```diff
@@ -0,0 +1,32 @@
+package Duke.Tasks;
+
+import Duke.Tool.Storage;
+import Duke.Tool.TaskList;
+import Duke.Ui;
+
+/**
+ * Represents the Find class
+ */
+public class Find extends Task {
+
+    protected String description;
+    /**
```

Comment:
Perhaps it would be better for the readability of the code if you left a line space between the end of the field declarations and the constructor?